lib/collection/src/operations/cluster_ops.rs

```diff
@@ -108,6 +110,7 @@ impl Validate for ClusterOperations {
             ClusterOperations::StartResharding(op) => op.validate(),
             ClusterOperations::AbortResharding(op) => op.validate(),
             ClusterOperations::FinishResharding(op) => op.validate(),
+            ClusterOperations::FinishMigratePoints(op) => op.validate(),
             ClusterOperations::CommitReadHashRing(op) => op.validate(),
             ClusterOperations::CommitWriteHashRing(op) => op.validate(),
```

Comment:
Not strictly related to this PR, but: I feel like it may be somewhat unclear (to users?) this belongs to resharding. How do you feel about changing naming a bit to make that more obvious?

Reply:
I though about it, but these are "internal" APIs anyway. We won't expose them in OpenAPI.

Reply:
I did mean for us.

Reply:
I propose to add some documentation comments to these variants. I don't want to change names, cause they already are pretty verbose and sticking `Resharding` in front of them would only make them more annoying. :/

Reply:
Sounds good.